src/geotiff.d.ts

```diff
@@ -0,0 +1,90 @@
+export declare class GeoTIFF {
+  getImage(index?: number): Promise<GeoTIFF.GeoTIFFImage>;
+  parseFileDirectoryAt(offset: number): Promise<GeoTIFF.ImageFileDirectory>;
+  readRasters(options?: GeoTIFF.RasterOptions): Promise<TypedArray>;
```

Comment:
```suggestion
```

I made this mistake when creating these types originally for Viv. This method does not exist on `GeoTIFF`.